pgpy/types.py

```diff
@@ -59,7 +59,7 @@ class Armorable(six.with_metaclass(abc.ABCMeta)):
     __armor_regex = re.compile(r"""# This capture group is optional because it will only be present in signed cleartext messages
                          (^-{5}BEGIN\ PGP\ SIGNED\ MESSAGE-{5}(?:\r?\n)
                           (Hash:\ (?P<hashes>[A-Za-z0-9\-,]+)(?:\r?\n){2})?
-                          (?P<cleartext>(.*\r?\n)+?)(?:\r?\n)?
+                          (?P<cleartext>(\r?\n|.)+?)(?:\r?\n)?
```

Comment:
I think this should be `(?P<cleartext>(.*\r?\n)+?)(?:\r?\n)`. `(?P<cleartext>(\r?\n|.)+?)(?:\r?\n)?` seems to capture the second newline instead of the first (so it gives an identical result), but it looks like it might be a bit more work for the regex engine

(see https://www.debuggex.com/r/pDfhxbanDJDKYgwN vs https://www.debuggex.com/r/GYgs7MaYet3fTNX8)